pkg/controller/nodenetworkconfigurationpolicy/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -196,6 +166,19 @@ func (r *ReconcileNodeNetworkConfigurationPolicy) Reconcile(request reconcile.Re
 	}
 
 	conditionsManager := conditions.NewManager(r.client, nodeName, *instance)
+	selectors := selectors.New(r.client, nodeName, *instance)
```

Comment:
The naming is confusing. We don't get a new selector here. Can we leverage the `Resource` struct here?